File: dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinTecsUserBackofficeSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinTecsUserBackofficeSDK.test();
      equal(true, null != testsdk);
    });
  });
}

File: lua/test/exists_test.lua
-- BluefinTecsUserBackoffice SDK exists test

local sdk = require("bluefin-tecs-user-backoffice_sdk")

describe("BluefinTecsUserBackofficeSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
